calcom/cal.com
# feat: Routing Forms: Ability to open dropdown by pressing "Enter"

## What does this PR do?
Earlier on using tab button for shifting from one to another fields, select field does not shows any options, we have to click on it to list options. Now on this PR, options list for select/multi-select fields shows automatically focusing on them.

Fixes #9312 

![feature (1)](https://github.com/calcom/cal.com/assets/77577810/29d99a31-de07-46e6-a5d9-73fef40204b8)


**Environment**: Staging(main branch) / Production

## Type of change
- New feature (non-breaking change which adds functionality)]

## How should this be tested?
1 create a new event
2 go to advanced
3 add a new question
4 chose the question as: select or multiselect and then save
5 Now as a guest book a meeting, and fill all the required fields and use tab buton for shifting from one input field to another 
6 On shifting to select field you will see options list showing on it's own without clicking on it
